modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/Filters.tsx

```diff
@@ -421,6 +408,60 @@ function Filters({
 		});
 	};
 
+	const onEdit = ({item}: {item: IFilter}) => {
+		if (
+			item.filterType === EFilterType.CLIENT_EXTENSION &&
+			!filterClientExtensionRenderers.length
+		) {
+			noFilterClientExtensionsAvailableModal();
+		}
+		else {
+			setActiveMode(FILTER_MODE.EDITION);
+			setActiveFilter(item);
+		}
+	};
+
+	const onUpdateStatus = async (item: IFilter) => {
```

Comment:
do we need the prefix `on` here? This sounds more just `updateActive` as well right?